subscription/notify-lambda/requirements.txt

```diff
@@ -1 +1,2 @@
 requests
+
```

Comment:
if you move `insert_final_newline = true` to the `[*]` section of ../.editorconfig then you might get this for free. You could also add a rule for .txt.

Reply:
I moved the rule to the [*] section.